gambit/README.md

```diff
@@ -0,0 +1,35 @@
+# Gambit MIMIC Integration
+
+## Overview
+
+Gambit MIMIC is a dynamic, real-time, high-performance SNMP, Netflow, sFlow Simulator. All Netflow and sFlow exports are recorded by the simulator and replayed with flexible customization toward Datadog agent(s) and presented with all existing available metrics of Datadog, whether built-in or Datadog marketplace integrated.
+
+## Setup
+
+### Installation
+
+Follow the configuration instructions below.
+
+### Configuration
+
+1. If you are already using Datadog and have the Datadog Agent installed, configure Gambit MIMIC agent to export Netflow and sFlow toward the Datadog Agent.
+
+### Validation
+
+## Data Collected
+
+### Metrics
+
+Gambit MIMIC is not limited to a particular set of metrics in Datadog. All existing or future avaiable metrics, if recorded by MIMIC, will be exported to Datadog agent.
+
+### Events
+
+## Troubleshooting
+
+Need help? Contact [Gambit MIMIC support][1].
```

Comment:
I assume this link should point to support@gambitcomm.com, not the overview website. Please adjust.